File: Entry.java
public class Entry {
    String date;
    String type;
    String stock;
    String amount;
    String price;
    String fees = "0";
    String split = "1";
    String dividend;

    public void printString() {
        System.out.println("date:"+this.date +", type:"+ this.type +", stock:"+ this.stock +", amount:"+ this.amount +", price:"+ this.price +", div:"+ this.dividend);
    }

    public void reset() {
        this.date = "";
        this.type = "";
        this.stock = "";
        this.amount = "";
        this.price = "";
        this.dividend = "";
    }
}

File: Entry.py
class Entry:
    def __init__(self):
        self.date = ""
        self.type = ""
        self.stock = ""
        self.amount = ""
        self.price = ""
        self.fees = "0"
        self.split = "1"
        self.dividend = ""

    def print_string(self):
        print(f"date: {self.date}, type: {self.type}, stock: {self.stock}, amount: {self.amount}, price: {self.price}, div: {self.dividend}")

    def reset(self):
        self.date = ""
        self.type = ""
        self.stock = ""
        self.amount = ""
        self.price = ""
        self.dividend = ""
        self.split = "1"
